.eslintrc.base.js

```diff
@@ -50,11 +54,11 @@ module.exports = {
   overrides: [
     {
       files: ["*.ts", "*.tsx"],
-      parserOptions: { parser: "eslint-plugin-typescript/parser" },
+      parserOptions: { parser: "@typescript-eslint/parser" },
       rules: {
         // Force a conflict with Prettier in test-lint/typescript.js.
         // This is included in "plugin:typescript/recommended".
```

Comment:
But of a nit as its in a comment but:

plugin:@typescript-eslint/recommended

Reply:
That's important for maintenance! Thank you for advises.